charmcraft.yaml

```diff
@@ -17,7 +17,7 @@ parts:
 bases:
   - build-on:
       - name: ubuntu
-        channel: "20.04"
+        channel: "22.04"
```

Comment:
It's better to build on focal and run on focal/jammy, or switch it to a 'binary' charm, and build for focal and jammy separately.  If you build on jammy (py3.10), then the py3.8 on focal *may* not work.  It's better to build on a py3.8 and then hopefully run on the py3.10.